privacy/index.md

```diff
@@ -35,8 +35,9 @@ In addition, you may elect to provide information to EFF through Certbot, such a
 
 EFF's Certbot Privacy Policy may change from time to time. However, any revised privacy policy will be consistent with EFF's mission. 
 
-*Updated on June 7th, 2017 to clarify which values are collected in the User Agent string.*
```

Comment:
Do we want to delete this? Other EFF [privacy](https://www.eff.org/policy) [policies](https://www.eff.org/code/privacy/policy) keep the summary of previous changes around.